libs/feature/search/src/lib/utils/service/fields.ts

```diff
@@ -322,3 +386,106 @@ export class OwnerSearchField extends SimpleSearchField {
     return of([])
   }
 }
+
+// ResourceContactField
+// contact of the data not metadata
+// data is the resource
+export class ContactField extends SimpleSearchField {
+  protected repository = this.injector.get(RecordsRepositoryInterface)
+  protected searchApiService = this.injector.get(SearchApiService)
+
+  // FIXME: this is required to register runtime fields; abstract this as well
+  protected esService = this.injector.get(ElasticsearchService)
+  private langService = this.injector.get(LangService)
+  private platformService = this.injector.get(PlatformServiceInterface)
+  public esFieldName: string
+  private version: Observable<string> = this.platformService
+    .getApiVersion()
+    .pipe(map((version) => version))
+
+  constructor(public order: 'asc' | 'desc' = 'asc', public injector: Injector) {
+    super('OrgForResource', order, injector)
+    this.esFieldName = 'OrgForResourceObject.default'
+  }
+
+  getFiltersForValues(values: FieldValue[]): Observable<FieldFilters> {
+    return this.version.pipe(
+      switchMap((version) =>
+        version == '4.2.2'
+          ? super.getFiltersForValues(values)
```

Comment:
Both lines are exactly the same as the default value of `esFieldName` in `getFiltersForValues` is `this.esFieldName`